xbmc/settings/AdvancedSettings.h

```diff
@@ -100,6 +103,8 @@ struct RefreshVideoLatency
 
 typedef std::vector<TVShowRegexp> SETTINGS_TVSHOWLIST;
 
+typedef std::function<void()> AdvancedSettingsCallback;
```

Comment:
We prefer `using`, not `typedef`.

=> https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#127-type-aliases